examples/inbound-deliveries-to-sap/main.bal

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/time;
+import ballerina/xmldata;
+import ballerinax/s4hana.api_inbound_delivery_srv_0002 as inbound;
+
+configurable S4HANAClientConfig s4hanaClientConfig = ?;
+
+final inbound:Client inboundClient = check new (
+    config = {
+        auth: {
+            username: s4hanaconfig.username,
+            password: s4hanaconfig.password
+        }
+    },
+    hostname = s4hanaconfig.hostname
+);
+
+public function main() returns error? {
+
+    xml readXml = check io:fileReadXml("files/document.xml");
+    inbound_deliveries filereading = check xmldata:fromXml(readXml);
```

Comment:
```suggestion
    InbDeliveries deliveries = check xmldata:fromXml(readXml);
```
Type always starts with Upper case

Reply:
Updated in [Update reviewed changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/16/commits/8650b99db9b8e7a6078a7120025cc0bf81979c17)